inputFiles/wellbore/ThermoPoroElasticWellbore_base.xml

```diff
@@ -73,7 +73,8 @@
     <!-- SPHINX_HeatCapacityProperties -->
     <SolidInternalEnergy
       name="rockInternalEnergy"
-      volumetricHeatCapacity="1.89e6"
+      referenceVolumetricHeatCapacity="1.89e6"
+      dVolumetricHeatCapacity_dTemperature="1e6"
```

Comment:
dVolumetricHeatCapacity_dTemperature for this case should be zero (default value)?
```suggestion

```

Reply:
Done